apps/src/code-studio/pd/application/teacher1920/Teacher1920Application.jsx

```diff
@@ -21,6 +21,17 @@ export default class Teacher1920Application extends FormController {
 
   static sessionStorageKey = 'Teacher1920Application';
 
+  componentDidMount() {
+    window.addEventListener('beforeunload', event => {
+      if (!this.state.submitting) {
+        event.preventDefault();
+        const customWarning = `Are you sure? Your application may not be saved.`;
+        event.returnValue = customWarning;
+        return customWarning;
```

Comment:
Are there browsers that do require the return even when `returnValue` is set?

Reply:
Ah, you're totally right and we don't need to do this! In fact, it looks like it [breaks this feature in IE11](https://stackoverflow.com/questions/34110931/beforeunload-on-ie-11-do-not-prompt-user-does-not-work).  I misread the [MDN example](https://developer.mozilla.org/en-US/docs/Web/API/WindowEventHandlers/onbeforeunload#Example) when throwing this together.  Thanks for catching!